Sources/BSON/BSONValue.swift

```diff
@@ -18,7 +18,7 @@ internal protocol BSONValue: Codable {
     /// Initializes a corresponding `BSON` from the provided extendedJSON.
```

Comment:
this docstring can be updated now that this method is fully implemented.

Reply:
So this initializer is initializing an instance of the `BSONValue` itself, not a `BSON` (like what `read` is doing). So could we update the docstring to mention that?

Reply:
yep